Decred Wallet/Features/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  TransactionsViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+
+class TransactionsViewController: UIViewController {
+    @IBOutlet weak var headerTopConstraint: NSLayoutConstraint!
+    @IBOutlet weak var headerStackView: UIStackView!
+    @IBOutlet weak var headerBottomConstraint: NSLayoutConstraint!
```

Comment:
- Rename these to have a `pageHeader...` prefix.
- Add an empty line space after these to separate from the next set of declarations that are not related to the page header.